src/DynamicData/Cache/ObservableCacheEx.BindSort.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using DynamicData.Binding;
+
+namespace DynamicData;
+
+/// <summary>
+/// ObservableCache extensions for BindAndSort.
+/// </summary>
+public static partial class ObservableCacheEx
```

Comment:
Are we cool with having some operators be organized like this, with a partial class, and some not? Would it be better to just reorganize ALL of them in one big PR?

Reply:
I have been meaning to split these up for years but I have always been way too daunted to tackle in one hit.  So let's leave this here as a placeholder.  The we can either move gradually, or if anyone is brave enough to do it in one hit.

Reply:
I'm down either way, just want to be clear. I'll go ahead and plan to do a big PR to reorganize this, once my current plate is clear. It should be a really simple PR, just.... very large... and tedious.